lib/restaurant.rb

```diff
@@ -0,0 +1,40 @@
+class Restaurant
```

Comment:
Like that you have a Restaurant class to hold everything - makes it much easier to understand!